src/resonanceReconstruction/rmatrix/BoundaryCondition.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef NJOY_R2_RMATRIX_BOUNDARYCONDITION
+#define NJOY_R2_RMATRIX_BOUNDARYCONDITION
+
+// system includes
+#include <string>
+
+// other includes
+
+namespace njoy {
+namespace resonanceReconstruction {
+namespace rmatrix {
+
+  // boundary condition
+  using BoundaryCondition = double;
```

Comment:
An entire file just for this? Seems a bit overkill. Can't this be included elsewhere?